addons/mail/static/src/core/common/thread_model.js

```diff
@@ -419,15 +442,13 @@ export class Thread extends Record {
         return !this.messages.some((message) => !message.isEmpty);
     }
 
-    get offlineMembers() {
-        const orderedOnlineMembers = [];
-        for (const member of this.channelMembers) {
-            if (member.persona.im_status !== "online") {
-                orderedOnlineMembers.push(member);
-            }
-        }
-        return orderedOnlineMembers.sort((m1, m2) => (m1.persona.name < m2.persona.name ? -1 : 1));
-    }
+    offlineMembers = Record.many("ChannelMember", {
+        /** @this {import("models").Thread} */
+        compute() {
+            return this.channelMembers.filter((member) => member.persona?.im_status !== "online");
```

Comment:
persona is guarded here but not in online compute, I'm not sure which of them is correct but it should be consistent